services/movie.py

```diff
@@ -22,19 +26,22 @@ def get_movie_by_id(movie_id: int) -> Movie:
     return Movie.objects.get(id=movie_id)
 
 
-def create_movie(
-    movie_title: str,
-    movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
-) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
+def create_movie(movie_title: Any, movie_description: Any,
+                 genres_ids: Any, actors_ids: Any) -> Movie:
+    try:
+        with ((((transaction.atomic())))):
+            if not isinstance(genres_ids, list) or not all(
+                    isinstance(i, int) for i in genres_ids):
```

Comment:
The check of `genres_ids` is not enough. You should also check if `genres_ids` is not empty. This is a bug because if `genres_ids` is an empty list, the movie will be created without genres.